activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/ProcessVariablesHelper.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.activiti.cloud.services.rest.controllers;
+
+import java.text.MessageFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.TimeZone;
+
+import org.activiti.api.process.model.payloads.SetProcessVariablesPayload;
+import org.activiti.engine.ActivitiException;
+import org.activiti.spring.process.model.Extension;
+import org.activiti.spring.process.model.ProcessExtensionModel;
+import org.activiti.spring.process.model.VariableDefinition;
+import org.activiti.spring.process.variable.VariableValidationService;
+
+public class ProcessVariablesHelper  {
+   
+    private final Map<String, ProcessExtensionModel> processExtensionModelMap;
+    private final VariableValidationService variableValidationService;
+    private String dateTimeFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
+    private TimeZone timeZone = TimeZone.getTimeZone("UTC");
+ 
+    public ProcessVariablesHelper(Map<String, ProcessExtensionModel> processExtensionModelMap,
+                                  VariableValidationService variableValidationService) {
+        this.processExtensionModelMap = processExtensionModelMap;
+        this.variableValidationService = variableValidationService;
+    }
+    
+    public String getDateTimeFormat() {
+        return dateTimeFormat;
+    }
+    
+    public void setDateTimeFormat(String dateTimeFormat) {
```

Comment:
why get/set for dateTimeFormat and timeZone? Cannot they just be constant?

Reply:
I have changed this part anyway: using DateFormatterProvider...
Setting of format can be useful if we need a format different, and want to set it from configuration file, etc... But if we need to remove setter - I will remove it...
setZoneId I have removed (it seems that this is really not needed)